Source/Utils/WarpXMovingWindow.cpp

```diff
@@ -353,6 +353,55 @@ namespace
 
 }
 
+amrex::Real
+WarpX::CalculateMovingWindowShiftPerStep (amrex::Real dt)
+{
+    // Calculate the shift per step (same formula as used in MoveWindow)
```

Comment:
```suggestion
    // Calculate the shift per step
```
The formula does not exist anymore in `MoveWindow`, so this comment is confusing.
(Instead we now call `CalculateMovingWindowShiftPerStep` in `MoveWindow` ; but I don't think comments should point out where the function is called - or if they should, then we also need to say that this function is called in `MovingWindowAndGalileanDomainShift`